grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationshipAtom.java

```diff
@@ -261,7 +261,7 @@ public boolean isAllowedToFormRuleHead(){
         }
 
         //check roles are ok
```

Comment:
Roles are no longer Types and this map seems to be collecting Types. Maybe this comment is now misleading?